CONTRIBUTING.md

```diff
@@ -30,6 +30,13 @@ Contributors can be nominated as Collaborators by Steering Committee members. On
 
 Nominations for Collaborator status happen through the typical Steering Committee decision making process. That is, to nominate one or more Collaborators, Steering Committee then moves to interview the nominees. The nominee is approved or rejected following the same consensus seeking process for new team members.
 
+#### Code Owners
+Collaborators can become a Code Owner for a specific package(s). They have the ability to approve a pull request that has made changes in those package(s) and at least one Code Owner will be required for any package.
```

Comment:
Should we call out the expectations of a code owner here? See https://chromium.googlesource.com/chromium/src/+/master/docs/code_reviews.md as an example

Reply:
@awentzel do we want more specificity?

Reply:
sounds good.

Reply:
Added a few bullet points for what is expected.